dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -1141,34 +1260,43 @@ EditTypeDlg_WriteClass(HWND hwndDlg, PEDITTYPE_DIALOG pEditType,
             if (RegCreateKeyExW(hVerbKey, L"command", 0, NULL, 0, KEY_WRITE, NULL,
                                 &hCommandKey, NULL) == ERROR_SUCCESS)
             {
-                // write the default value
-                DWORD dwSize = (value.GetLength() + 1) * sizeof(WCHAR);
-                RegSetValueExW(hCommandKey, NULL, 0, REG_EXPAND_SZ, LPBYTE(LPCWSTR(value)), dwSize);
-
+                DWORD dwSize = (cmd.GetLength() + 1) * sizeof(WCHAR);
+                DWORD dwType = REG_SZ;
+                LPCWSTR exp;
+                if ((exp = StrChrW(cmd, '%')) && StrChrW(exp + 1, '%'))
```

Comment:
Considering `cmd` is a `CStringW`, why are you using `StrChrW` here, and not [CStringT::Find](https://learn.microsoft.com/en-us/cpp/atl-mfc-shared/reference/cstringt-class?view=msvc-170#find) ?